discord/__init__.py

```diff
@@ -19,6 +19,18 @@
 
 __path__ = __import__('pkgutil').extend_path(__path__, __name__)
 
+import platform
+import asyncio
+
+# Required if proxies are being used on Windows to prevent 'Command is incorrect' error
+if platform.system() == "Windows":
+    asyncio.set_event_loop_policy(policy=asyncio.WindowsSelectorEventLoopPolicy())
+
+# Required for the captcha solver if we use the cheaper capmonster over 2captcha.com via host-entry edit
```

Comment:
This should be happening inside the captcha solver, since it isn't needed for most users.